src/components/Form.js

```diff
@@ -269,12 +269,24 @@ class Form extends React.Component {
                 .value() || '';
 
             return React.cloneElement(child, {
-                ref: node => this.inputRefs[inputID] = node,
+                ref: (node) => {
+                    this.inputRefs[inputID] = node;
+
+                    // Call the original ref, if any
+                    const {ref} = child;
+                    if (_.isFunction(ref)) {
```

Comment:
We do this check in many places, where we check if something is a function and if it is we invoke it. Should we replace these instances with `Func.invoke` like `Func.invoke(ref)`? 
https://github.com/Expensify/expensify-common/blob/36d6f23/lib/Func.jsx#L13

Reply:
This is not necessary a repeated logic. Although `Func.invoke` makes the block of code in one line, it may be confusing; We will have to call `Func.invoke(undefined, [ref])`. Notice that the argument must be an array which is an easy thing to miss.

Reply:
I won't block on this, since this duplication is already here, but here's why I don't think I agree, just for context
- Just in this PR, we could replace 3 lines of code with 1, in 5 places, so I don't see why this wouldn't be considered repeated logic
- We'll have to call `Func.invoke(red, [node])` in this case, the params being in an array is as easy to miss as any other piece of code, like passing `node` to `ref(node)`. Passing arguments to a function in an array is pretty standard in JS when using `Function.prototype.apply`